costmap_cspace/include/costmap_cspace/pointcloud_accumulator.h

```diff
@@ -108,6 +108,11 @@ class PointcloudAccumurator
   ros::Duration time_to_hold_;
   std::list<Points> points_;
 };
+
+// to keep backward compatibility with code base using PointcloudAccumurator
+template <typename T>
+using PointcloudAccumurator = PointcloudAccumulator<T>;
```

Comment:
It would be better to mark as deprecated using C++14 `[[deprecated]]` attribute.
https://en.cppreference.com/w/cpp/language/attributes/deprecated